src/apps/competitions/models.py

```diff
@@ -538,6 +542,13 @@ def start(self, tasks=None):
         from .tasks import run_submission
         run_submission(self.pk, tasks=tasks)
 
+    def run(self):
+        # get tasks from the phase
+        tasks = self.phase.tasks.all()
+        # start submission providing the tasks
+        self.start(tasks=tasks)
+        return self
+
```

Comment:
Why is this change needed?

Reply:
this change is for running the submission inside run_submission function. self.start forwards the submission for running